bigchaindb/commands/utils.py

```diff
@@ -151,6 +158,10 @@ def mongodb_host(host):
                          help='Specify the location of the configuration file '
                               '(use "-" for stdout)')
 
+base_parser.add_argument('-l', '--log-level',
+                         choices=['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL'],
+                         help='Log level')
+
```

Comment:
Thanks for copying my comment over.

I don't see any changes to the docs in this pull request, or any mention of the new`-l` option in the master branch. Am I missing something?